docs/API_README.md

```diff
@@ -0,0 +1,142 @@
+## API Docs
+
+**Interface to sign in**
+
+| URI      | Method | Authorization |
+|----------|--------|---------------|
+|`/token/` | `POST` | not required  |
+
+
+Params:
```

Comment:
Also, could you replace query params to JSON data in POST queries to be more safety